editoast/src/views/v2/train_schedule.rs

```diff
@@ -676,6 +709,35 @@ mod tests {
         assert!(response.status().is_client_error());
     }
 
+    #[rstest]
+    async fn get_batch_trainschedule(
```

Comment:
Let's test a real batch, with at least more than one train? Because `get` with a single train is already tested.

Reply:
Done